extensions/sql-migration/README.md

```diff
@@ -1,11 +1,12 @@
 # Azure SQL Migration
 The [Azure SQL Migration extension for Azure Data Studio](https://docs.microsoft.com/sql/azure-data-studio/extensions/azure-sql-migration-extension) brings together a simplified assessment, recommendation, and migration experience that delivers the following capabilities:
-- A responsive user interface that provides you with an end-to-end migration experience that starts with a migration readiness assessment, SKU recommendation (based on performance data), and finalizes with the actual migration to Azure SQL.
-- An enhanced assessment mechanism that can evaluate SQL Server instances, identifying databases that are ready for migration to the different Azure SQL targets.
-- A SKU recommendation engine (Preview) that collects performance data from the source SQL Server instance on-premises, generating right-sized SKU recommendations based on your Azure SQL target.
+- A responsive UI for an end-to-end migration experience. The extension starts with a migration readiness assessment and SKU recommendation (preview) (based on performance data).
```

Comment:
I just noticed this now, maybe we should reword SKU recommendation to right-sized recommendation? It seems elsewhere in the extension/documentation we've moved away from "SKU"

Reply:
Done

Reply:
I'm still seeing SKU here. is that what we want?